easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2021b.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'kb-python'
+version = '0.27.3'
+
+homepage = 'https://www.kallistobus.tools/'
+description = """kallisto | bustools is a workflow for pre-processing single-cell RNA-seq data.
+ Pre-processing single-cell RNA-seq involves: (1) association of reads with their cells of origin,
+ (2) collapsing of reads according to unique molecular identifiers (UMIs), and (3) generation of
+ gene or feature counts from the reads to generate a cell x gene matrix."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('h5py', '3.6.0'),
+    ('IPython', '7.26.0'),
+    ('loompy', '3.0.7'),
+    ('Pysam', '0.17.0'),
+    ('Python', '3.9.6'),
+    ('scanpy', '1.8.2'),
+    ('scikit-learn', '1.0.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('shortuuid', '1.0.11', {
+        'checksums': ['fc75f2615914815a8e4cb1501b3a513745cb66ef0fd5fc6fb9f8c3fa3481f789'],
+    }),
+    ('ngs-tools', '1.8.3', {
+        'checksums': ['c50e36105654cdea14b4c7971994a3e710157663aaf0fd72c4d83a486f12cdb0'],
+    }),
+    ('tenacity', '8.1.0', {
+        'checksums': ['e48c437fdf9340f5666b92cd7990e96bc5fc955e1298baf4a907e3972067a445'],
+    }),
+    ('plotly', '5.13.0', {
+        'checksums': ['81a3aae4021d5ab91790fc71c3433791f41bfc71586e857f7777f429a955039a'],
+    }),
+    (name, version, {
+        'sources': ['kb_python-%(version)s.tar.gz'],
+        'checksums': ['dc98f6ceb4402d666b7e0d19be17c63d33e8b710a35cdc33de7c0f457122f43f'],
+    }),
+]
+
```

One other thing, it would be nice to have a sanity check command for the installed executable. Since just `kb info` or `kb --help` returns with an exit code 1, we could grep the version number from `kb info` and check it that has the expected value:
```suggestion

sanity_check_commands = ['kb info | grep "kb_python %(version)s"']

```